app/models/forms/contribution.rb

```diff
@@ -84,11 +84,12 @@ def insert_embargo_date
     @tufts_pdf.embargo_note = (Time.zone.now + embargo_note.to_i.months).iso8601 unless (embargo_note || '0').eql? '0'
   end
 
-  def insert_rels_ext_relationships
-    return unless @tufts_pdf
-    @tufts_pdf.stored_collection_id = parent_collection
-    @tufts_pdf.rels_ext.serialize!
-  end
+  # def insert_rels_ext_relationships
```

Comment:
Is there a ticket for this todo? If so, can we just drop this code?

Reply:
I made issue #256 for this, I'll just drop this method from the code